shared/common-adapters/labeled-input.tsx

```diff
@@ -45,8 +45,12 @@ const ReflessLabeledInput = (props: Props & RefProps) => {
     [value, onChangeText]
   )
 
+  // If we're uncontrolled its possible its been injected into unbeknownst to us
```

Comment:
to repro show the compnent, inject into, then focus/ blur and it'll overlap as before it had no idea there was text